bread/formatters.py

```diff
@@ -149,19 +137,26 @@ def as_video(value):
     )
 
 
-MODELFIELD_FORMATING_HELPERS = {
-    None: lambda a: a,
-    models.EmailField: as_email,
-    models.FileField: as_download,
-    models.URLField: as_url,
-    models.TextField: as_text,
-    models.TimeField: as_time,
-    models.DateTimeField: as_datetime,
-    CountryField: as_countries,
-}
-
 CONSTANTS = {
     None: getattr(settings, "HTML_NONE", app_settings.HTML_NONE),
     True: getattr(settings, "HTML_TRUE", app_settings.HTML_TRUE),
     False: getattr(settings, "HTML_FALSE", app_settings.HTML_FALSE),
 }
+
+
+# copied from https://github.com/django/django/blob/1f9874d4ca3e7376036646aedf6ac3060f22fd69/django/utils/html.py
+# because older versions only have this as a local function available
```

Comment:
Older versions == Django 3.2 does not have this function accessible from outside of the urlize funtion.